models/ParagraphStyle/index.d.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+declare class ParagraphStyle {
+    _class: 'paragraphStyle';
+    alignment: number;
```

Comment:
Should this use an enum like we do here: https://github.com/amzn/sketch-constructor/blob/master/models/BorderOptions/index.d.ts#L1

Reply:
yes, you're right :)

Reply:
fixed